src/Servers/HttpSys/test/FunctionalTests/ResponseBodyTests.cs

```diff
@@ -133,6 +133,41 @@ public async Task ResponseBody_WriteNoHeaders_SetsChunked()
         }
     }
 
+    [ConditionalTheory]
+    [InlineData(true)]
+    [InlineData(false)]
+    public async Task ResponseBody_WriteNoHeaders_SetsChunked_LargeBody(bool enableKernelBuffering)
+    {
+        const int WriteSize = 1024 * 1024;
+        const int NumWrites = 32;
+
+        string address;
+        using (Utilities.CreateHttpServer(
+            baseAddress: out address,
+            configureOptions: options => { options.EnableKernelResponseBuffering = enableKernelBuffering; },
```

Comment:
I think the unit test should set this value via `AppContext`. That's what a real .NET 7 app would have to do.

Reply:
Unit tests cannot set AppContext, it's static shared state that will mess up other tests.

Reply:
ok